package.json

```diff
@@ -7,10 +7,10 @@
     "license": "LGPL-3.0",
     "repository": "https://github.com/cookiecord/cookiecord",
     "dependencies": {
-        "chokidar": "^3.3.1",
-        "discord.js": "discordjs/discord.js#d0bc4d7ff037035d4f315a0286b06f0a1ade29c1",
+        "chokidar": "^3.5.2",
```

Comment:
All these package versions have been proven to be reliable, no need to update if it ain't broken.

Reply:
that is not true, stable released are made to resolve issues in previous version, if you use a commit as package, it will cause unnecessary issues at developer end. Causing type mismatch between versions.

Reply:
No, they are expected to use the dependency without redeclaring it.

Reply:
However, I don't see a reason for not using stable version?

Reply:
I think there was a weird bug on `#stable` a while ago so it has been like this ever since. It also prevents accidental changes in `yarn.lock` which happened once a while ago.